### ISSUE TITLE
Missing/broken images during browser and Inspect preview

### ISSUE THREAD

**Issue by line49** [edge-reflow/issues#101]
Situation:
Import from PSD brought over the desired content, including images.
Images were (of course) generated during PSD import as a result of naming the images, in this case with PNG extension.
Images existed in correct place in Windows file system (img folder) and were visible in Reflow.

Issue:
Images were NOT visible when previewed in Chrome or Edge Inspect satellite devices.
I had to manually copy and re-import images to get them to show up.

In other words, there was no problem with the Windows environment or anything third-party; somehow the preview mode did not reference the image files properly when they were first pulled over from Photoshop. However replacing the images (with themselves, using a _different filename_) "solved" the problem.

Example:
Unfortunately I did not take a screen grab.

**Comment by vmitnick** [comment]
Did this just happen once, or is it reproducible? If it happens again, it would be very useful for us to get a copy of the failing Reflow project.

**Comment by jtlafferty** [comment]
Or, I would love to see the PSD that was used. Please send it to tan@adobe.com. Thanks!

**Comment by jtlafferty** [comment]
I received the PSD and am looking into trying to reproduce the problem.

**Comment by jtlafferty** [comment]
I haven't reproduced this issue on Windows yet with the customer's PSD, but, I've pinged them for more information.

**Comment by line49** [comment]
Hi Joan,

Yes, things like this never reproduce well, do they. Here are my answers. In short, there's nothing new or unusual that I can offer you in terms of a clue.

1) Can you go to Help -> About and tell me what version of Reflow you are running.

```
        Version 0.31.13658 (the latest version).
```

3) Also, did you preview in Chrome via the "View-> Preview in Chrome" menu item, or did you just go and find the html file and launch that in Chrome.

```
     I used "View-> Preview in Chrome".
     [I believe I probably also manually browsed to the Reflow HTML file and confirmed that the issue persisted that way as well] 
```

2) In your bug description, you say, "I had to manually copy and re-import images to get them to show up.", where did you copy images to get them to show up? 

```
       In Windows, I browsed to the PNG files which had been created successfully by the "Generate Reflow project" function in Photoshop CC (also latest version). I then literally copied them to the same folder by right-clicking and dragging to create duplicate files named "logo.png - Copy" etc.  I then used the 'replace asset' function in Reflow to replace the original images with these clones.

        I did this because I had a suspicion that I could solve the issue by replacing the original files with new copies (of the same files) with different filenames, thus forcing Reflow to "refresh" the reference/linking to those files. It worked. 

         As a test, I first "replaced" one of the files with the actual same original file, meaning that the filename did not change. When I did that, the asset linking was not 'refreshed' and the image continued to appear broken. Only after using a cloned copy with an amended filename did the refresh work.
```

**Comment by jtlafferty** [comment]
I went back to this bug again and tried more things to try and reproduce it. I used the customer's PSD and a few others. I cannot get this to reproduce. :(
